### DISCUSSION TITLE
DP2SceneGraphWizard

### DISCUSSION THREAD

**Discussion by fronderblash** [rrirower/DP2SceneGraphWizard#94] [Ideas]
hello 
please can will use DP2SceneGraphWizard for amazon apk file 
![ghvg](https://github.com/rrirower/DP2SceneGraphWizard/assets/141512200/d714298c-9556-4aec-b535-f4849547f6db)

**Comment by rrirower** [comment]
I have no plans to support Amazon channels through the Wizard project.  The ability to create an Amazon channel is already available.  [Amazon Creator](https://creator.amazon.com) is the equivalent of Roku's Direct Publisher.  Using the [Channel Feed Manager](https://github.com/rrirower/Channel-Feed-Manager) suite of tools on this site, you can export your feed file to an Amazon Creator format.

**Comment by axelctv** [comment]
Keep in mind that Amazon decided a couple of months ago to drop Android, and they are moving all apps to their own coding language called Vega OS  - so APK packages will not work/be accepted soon. This is one of the hundred of articles taking about this :) https://www.theverge.com/2023/11/14/23954333/amazon-ditching-android-fire-tv-echo-show
